hack/upgrade-test.sh

```diff
@@ -205,6 +205,11 @@ echo "----- Images before upgrade"
 ${CMD} get deployments -n ${HCO_NAMESPACE} -o yaml | grep image | grep -v imagePullPolicy
 ${CMD} get pod $HCO_CATALOGSOURCE_POD -n ${HCO_CATALOG_NAMESPACE} -o yaml | grep image | grep -v imagePullPolicy
 
+
+echo "----- HCO deployOVS annotation before the upgrade"
+PREVIOUS_OVS_ANNOTATION=$(${CMD} get HyperConverged kubevirt-hyperconverged -o jsonpath='{.metadata.annotations.deployOVS}')
```

Comment:
question: is this test performed on various versions? meaning, do we check upgrade from to all supported versions?

Reply:
We have several upgrade test lanes:

- ci/prow/hco-e2e-upgrade[aws|azure]  - tests an upgrade from current version to the code in the current PR.
- ci/prow/hco-e2e-upgrade-prev[aws|azure] - tests an upgrade from the previous minor version (Y-1) directly to the content of the PR.

`PREVIOUS_OVS_ANNOTATION` is the annotation in the cluster, prior to the upgrade.